_episodes/13-looking-up-data.md

```diff
@@ -79,7 +79,7 @@ There are a few services where you can find an OpenRefine Reconciliation option
 
 In other cases people have built reconciliation applications for a specific service which you can download and run yourself. These vary in how they work, and what it takes to run them locally. For example there are multiple reconciliation applications for VIAF. Even for the same service (e.g. VIAF) different reconciliation applications (written by different people)  can work in different ways and potentially give different results - so caveat emptor!
 
-One of the most common ways of using the reconciliation option in OpenRefine is with an extension (see below for more on extensions to OpenRefine) can use linked data sources for reconciliation. The extension is called "RDF Refine" and can be downloaded from [http://refine.deri.ie](http://refine.deri.ie).
+One of the most common ways of using the reconciliation option in OpenRefine is with an extension (see below for more on extensions to OpenRefine) can use linked data sources for reconciliation. The extension is called "RDF Refine" and can be downloaded from [http://refine.deri.ie](https://web.archive.org/web/20180112163745/http://refine.deri.ie/) (Available at Internet Archive Wayback Machine).
```

Comment:
The RDF extension is now being supported by Stuart Kenney and should be downloaded from https://github.com/stkenny/grefine-rdf-extension/releases

Reply:
Thanks! Pointed to the Stuart Kenney link.